bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/bstar_tW_Left_M1200/bstar_tW_Left_M1200_proc_card.dat

```diff
@@ -0,0 +1,25 @@
+set group_subprocesses Auto
+set ignore_six_quark_processes False
+set loop_optimized_output True
+set complex_mass_scheme False
+import model sm
```

Comment:
can you please remove the multiple import statements and the multiple definitions of proton and jet?
https://twiki.cern.ch/twiki/bin/viewauth/CMS/Moriond18MC#Checks_to_the_datacards_BEFORE_T